package.json

```diff
@@ -67,38 +68,39 @@
     ]
   },
   "devDependencies": {
-    "@babel/core": "7.9.6",
+    "@babel/core": "~7.9.0",
     "@babel/preset-typescript": "7.9.0",
     "@types/jest": "25.2.1",
-    "@types/react": "^16.9.11",
-    "@types/react-native": "^0.60.22",
+    "@types/react": "~16.9.35",
+    "@types/react-native": "~0.63.2",
     "@types/react-native-communications": "2.2.1",
     "@types/react-test-renderer": "16.9.2",
     "@types/uuid": "3.4.9",
     "babel-core": "7.0.0-bridge.0",
     "babel-jest": "25.5.1",
-    "babel-preset-expo": "^8.1.0",
-    "expo": "^37.0.0",
-    "expo-constants": "~9.0.0",
-    "expo-image-picker": "~8.1.0",
-    "expo-location": "~8.1.0",
-    "expo-permissions": "~8.1.0",
+    "babel-preset-expo": "8.3.0",
+    "expo": "^41.0.0",
+    "expo-app-loading": "1.0.3",
+    "expo-constants": "~10.1.3",
+    "expo-image-picker": "~10.1.4",
+    "expo-location": "~12.0.4",
+    "expo-permissions": "~12.0.1",
```

Comment:
Please consider moving the example dependencies to a separate example app project. otherwise it's difficult to have a good idea of what this package is putting in people's code

Reply:
I do agree with this sentiment.  It's not high on my list, but definitely something I wanna do soon.

Reply:
@shamilovtim This was done and released as [beta-2](https://github.com/FaridSafi/react-native-gifted-chat/releases/tag/v1.0.0-beta-2)